projects/igniteui-angular/src/lib/grids/cell.component.ts

```diff
@@ -893,6 +893,7 @@ export class IgxGridCellComponent implements OnInit, OnChanges, OnDestroy {
         }
 
         // TODO: to be deleted when onFocusChange event is removed #4054
+        isWarningShown(showMessage('onFocusChange event is deprecated. Use onGridKeydown event instead.', warningShown));
```

Comment:
You can put the event name in quotes to comply with the previous deprecation messages.